zerver/tests/test_signup.py

```diff
@@ -608,4 +608,50 @@ def test_create_realm(self):
             self.assertTrue(result["Location"].endswith("/invite/"))
 
             result = self.client.get(result["Location"])
-            self.assert_in_response("You're the first one here!", result)
+            self.assertIn("You're the first one here!", result.content)
+
+    def test_create_realm_with_subdomain(self):
+        # type: () -> None
+        username = "user1"
+        password = "test"
+        domain = "test.com"
+        email = "user1@test.com"
+        subdomain = "test"
+        realm_name = "Test"
+
+        # Make sure the realm does not exist
+        self.assertIsNone(get_realm(domain))
+        with self.settings(REALMS_HAVE_SUBDOMAINS=True), self.settings(OPEN_REALM_CREATION=True):
+            # Create new realm with the email
+            result = self.client.post('/create_realm/', {'email': email})
+            self.assertEquals(result.status_code, 302)
+            self.assertTrue(result["Location"].endswith(
+                    "/accounts/send_confirm/%s@%s" % (username, domain)))
+            result = self.client.get(result["Location"])
+            self.assertIn("Check your email so we can get started.", result.content)
```

Comment:
Use `self.assert_in_response("Check your email so we can get started.", result)`